components/table/hooks/useFilter/FilterDropdown.tsx

```diff
@@ -510,9 +567,19 @@ function FilterDropdown<RecordType>(props: FilterDropdownProps<RecordType>) {
   }
 
   const menu = () => (
-    <FilterDropdownMenuWrapper className={`${prefixCls}-dropdown`}>
-      {dropdownContent}
-    </FilterDropdownMenuWrapper>
+    <FocusLock
+      onActivation={() => {
+        focusFirstFocusableElement();
+      }}
+      onDeactivation={() => {
+        triggerBtnRef.current?.focus();
+      }}
+      autoFocus={false}
```

Comment:
I replaced this with `focusFirstFocusableElement` function, because there are some issues on the page `http://localhost:8001/components/table` where there are a lot of components, and focus is called too fast before dropdown is rendered, it results in a page to be scrolled to the top. We need at least zero timeout here, but react-focus-lock doesn't allow to add it.